guides/source/active_record_callbacks.md

```diff
@@ -585,6 +573,18 @@ irb> @user.save # updating @user
 User was saved to database
 ```
 
+WARNING. In the context of a single transaction, if you interact with multiple
```

Comment:
This is maybe for another PR, but I find these huge walls of text (especially in a note/warning block) with no sample code to be difficult to catch.

Reply:
I agree.